### ISSUE TITLE
Command "/w" missing arguments

### ISSUE THREAD

**Issue by Aareon** [AnonymousDapper/rwci-client#2]
```
TypeError: command_w() missing 1 required positional argument: 'recipient'
```
Should this return some sort of "usage" message rather than an exception?

**Comment by AnonymousDapper** [comment]
@Aareon Yes it should, but I just never got around to adding that. I hadn't thought of a good, easy way to do that. Maybe argparse

**Comment by AnonymousDapper** [comment]
This was actually fixed in rewrite with 5da5b93
